clients/venice-client/src/main/java/com/linkedin/venice/fastclient/RetriableAvroGenericStoreClient.java

```diff
@@ -40,7 +34,7 @@ public class RetriableAvroGenericStoreClient<K, V> extends DelegatingAvroStoreCl
   private static final Logger LOGGER = LogManager.getLogger(RetriableAvroGenericStoreClient.class);
 
   public RetriableAvroGenericStoreClient(InternalAvroStoreClient<K, V> delegate, ClientConfig clientConfig) {
-    super(delegate);
+    super(delegate, clientConfig);
```

Comment:
No retry for `compute` APIs?

Reply:
I think even thin-client doesn't do retries. So, I kept consistency with that. I can add it if it was a miss

Reply:
Retry in fast-client is a must since it is trying to mimic the retry in VeniceRouter.
Without retry, the p99 latency will be bad.

In thin-client, the retry is mainly for error retry and D2 has the client retry feature for all kinds of request by default, and d2 client retry is latency-based.

Reply:
Added the retries and the unit tests for them too